BitWatch/HomeViewController.swift

```diff
@@ -9,30 +9,18 @@
 import UIKit
 import BitWatchKit
 
-class HomeViewController: UIViewController {
 
+class HomeViewController: UIViewController, HomeVCDelegate{
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        // Do any additional setup after loading the view.
         let height = self.view.frame.size.height;
         let width = self.view.frame.size.width;
         
-        // Create the Home View
-        var homeView: UIView = UIView(frame: CGRectMake(0, (height / 2), width, 50));
-        homeView.backgroundColor = UIColor(white: 1.0, alpha: 1.0);
-        
-        // Create the "Begin" button
-        var beginButton: UIButton = UIButton(frame: CGRectMake(0, (height * 0.25), width, (height * 0.75)));
-            beginButton.setTitle("begin", forState:UIControlState.Normal);
-        
-        // Create the "Menu" button
-        var menuButton: UIButton = UIButton(frame: CGRectMake(width - 60, 10, 50, 50));
-        var menuImage: UIImage! = UIImage(named: "menu-icon");
-            menuButton.setBackgroundImage(menuImage, forState: UIControlState.Normal);
-        
-        homeView.addSubview(beginButton);
-        homeView.addSubview(menuButton);
+        self.view.backgroundColor = UIColor.whiteColor()
+        var homeView: HomeView = HomeView(frame: self.view.frame)
```

Comment:
I like to compartmentalize the whole view as it's own file. This way all the view logic and subviews don't clutter up the VC.